### DISCUSSION TITLE
Help with interface

### DISCUSSION THREAD

**Discussion by alexblazex** [mozman/ezdxf#1120] [Q&A]
Hello!

**Your library is really very useful and cool!!**

Can you please help me? 

There is a task: I want to use your interface to learn how to draw a circle of a certain diameter at the place where the right mouse button is pressed. 

I also added some controls to your _ezdxf/src/ezdxf/addons/drawing
/ fileqtviewer.py_

I've already managed to draw or write something. It's not difficult. However, this is not saved on the layout and when zooming with the wheel or mouse capture and moving the object, the dots or labels obviously remain in place and do not move with the drawing.  

Unfortunately, I couldn't figure out the documentation for drawing new objects (circle). How do I change the _ezdxf/src/ezdxf/addons/drawing/ fileqtviewer.py_
 to make my functionality work? 

Thank you so much in advance for your help! As you can see, I'm not good at python at all :(
![image](https://github.com/mozman/ezdxf/assets/76105841/72678318-701a-42a3-b15e-78250573cb1a)
![image](https://github.com/mozman/ezdxf/assets/76105841/5ec2bd0b-7a54-4d6e-bb42-f4d07aba6478)

This is an example of labels remaining in place when captured or scaled.

**Comment by mozman** [comment]
Hi!

Sorry, I can't help. The `ezdxf` library is not designed and optimized to be used as a CAD kernel.

**Reply by alexblazex** [reply]
Hello!

Thanks a lot for the answer, although I'm sorry to hear it :)
